mlflow/sklearn/__init__.py

```diff
@@ -37,6 +37,7 @@
     INPUT_EXAMPLE_SAMPLE_ROWS,
     resolve_input_example_and_signature,
     _get_new_training_session_class,
+    AutologgingBatchingClient,
```

Comment:
Incorporate `AutologgingBatchingClient` into the `mlflow.sklearn` autologging flavor.